underscore.js

```diff
@@ -1446,6 +1453,13 @@
     return _.isFunction(value) ? value.call(object) : value;
   };
 
+  // Returns the, possibly deeply nested, value at `path` if defined.
+  // Otherwise, returns `fallback`.
+  _.get = function(object, path, fallback) {
+    var value = property(path)(object);
+    return value === void 0 ? fallback : value;
+  };
```

Comment:
I see. Yeah it make sense to start with `_.result` since it would be a superset of `_.get`'s functionality, and we can do it without committing to `_.set` and `_.unset` (We probably will need to update `_.has` as well).

Do you think we should aim for evaluating functions at every step a la https://github.com/lodash/lodash/commit/f65d36b3c1d87c95534e56fba98b576a452e8fb7 ?